blog/articles/set-up-a-local-knative-environment-up-in-no-time-flat.md

```diff
@@ -0,0 +1,310 @@
+---
+Title: 'How to set up a local Knative environment up in no time flat'
+Author: Leon Stigter
```

Comment:
The title sounds too link bate.

Why not leverage the term `kind`

How to use kind to setup Knative local environment with no DNS headaches

Reply:
I've updated the title (and filename) to `How to set up a local Knative environment with KinD and without DNS headaches`